py/dml/messages.py

```diff
@@ -1895,6 +1895,44 @@ def __init__(self, site, old, new, maybe_intended):
                       if maybe_intended else '')
         DMLWarning.__init__(self, site, old, new, suggestion)
 
+class WLGROUPSASLEV(DMLWarning):
+    """
+    The specified log level of a `log` looks as though you meant to specify the
+    log groups instead. For example:
+    ```
+    log info, some_log_group: ...;
+    log info, (some_log_group | another_log_group), 1: ...;
```

Comment:
Note that we generally discourage `log info` on level 1 in production code (it's good for printf debugging only). So better to use e.g. `spec_viol` for the level 1 cases.